css-borders-4/Overview.bs

```diff
@@ -617,16 +617,29 @@ Initial: none
 Applies to: all elements
 Inherited: no
 Percentages: N/A
-Computed value: either 'none' or a list,
-	each item a pair of offsets (horizontal and vertical) from the element‘s box
-Animation type: by computed value
+Computed value: list, each item either 'none' or a pair of offsets
+  (horizontal and vertical) from the element‘s box
+Animation type: by computed value,
+    letting ''box-shadow-offset/none'' create a blank shadow
+    (''transparent 0 0 0 0'') when interpolated with non-'none' values
+    and appending blank shadows with a corresponding
```

Comment:
I think this should actually be handled in each longhand, i.e. the animation type of `box-shadow-offset` should just add `none` values, `box-shadow-blur` add `0`, etc.